brototype.js

```diff
@@ -53,14 +53,28 @@
         },
 
         "doYouEven": function(key, options) {
-            var optionsBro = Bro(options || {}),
-                bro = this.iCanHaz(key);
-            if (Bro(bro).isThatEvenAThing() === Bro.TOTALLY) {
-                optionsBro.iDontAlways('forSure').butWhenIdo();
-                return Bro.TOTALLY;
+            var optionsBro = Bro(options || {});
+            if (key instanceof Array) {
```

Comment:
Why not

    if (!(key instanceof Array)) {
        key = [key];
    }

Rather than the duplicated code?